src/components/Sidenav/Sidenav.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Sidenav = () => {
```

Comment:
Is this component used somewhere? If you're not planning on using then delete, but maybe you're planning to integrate later in your final project (in which case of course leave in).